mix.exs

```diff
@@ -2,7 +2,7 @@ defmodule StellarBase.MixProject do
   use Mix.Project
 
   @github_url "https://github.com/kommitters/stellar_base"
-  @version "0.3.0"
+  @version "1.0.0"
```

Comment:
Are we ready for a 1.0 version meaning it can be used in production environments?

Reply:
@luishurtado I'm not really sure 🤔 , let me think about it!!

Reply:
@luishurtado you're right! testing other scenarios before is important before releasing a stable version.

Closing this in favor of #130